### PR TITLE
Updated Documentation to be more explizit about authorizationContext …

### PR DESCRIPTION
**Pull request by JulianFeinauer**
…in Piggyback Commands

After a short discussion with @thjaeckle.
My ICLA for my eclipse account (j.feinauer@pragmaticminds.de) is filed.

### CODE REVIEW

File: documentation/src/main/resources/pages/ditto/basic-auth.md [hunk 1]
```diff
@@ -33,3 +33,13 @@ Authorization is implemented with an <a href="#" data-toggle="tooltip" data-orig
 (in API version 2).
 
 Please find details at [ACL](basic-acl.html) and [Policies](basic-policy.html).
+
+### Authorization Context in DevOps Commands
+
+An `authorizationContext` which is passed to [DevOps Commands](installation-operating.html#devops-commands) DevOps commands needs
+to be a subject known to dittos authentication. In the simplest case, its `nginx:{ditto-username}` where `{ditto-username}` is a ditto user
```
Comment (thjaeckle) [inline documentation/src/main/resources/pages/ditto/basic-auth.md]:
```suggestion
to be a subject known to Ditto's authentication. In the simplest case, it's `nginx:{username}` where `{username}` is a user 
```

File: documentation/src/main/resources/pages/ditto/basic-auth.md [hunk 2]
```diff
@@ -33,3 +33,13 @@ Authorization is implemented with an <a href="#" data-toggle="tooltip" data-orig
 (in API version 2).
 
 Please find details at [ACL](basic-acl.html) and [Policies](basic-policy.html).
+
+### Authorization Context in DevOps Commands
+
+An `authorizationContext` which is passed to [DevOps Commands](installation-operating.html#devops-commands) DevOps commands needs
+to be a subject known to dittos authentication. In the simplest case, its `nginx:{ditto-username}` where `{ditto-username}` is a ditto user 
+that is configured for basic auth. The `nginx:` prefix refers to basic authentication here.
```
Comment (thjaeckle) [inline documentation/src/main/resources/pages/ditto/basic-auth.md]:
```suggestion
that is configured for basic auth in the included nginx's `nginx.htpasswd` file (where the `nginx:` prefix comes from).
```

File: documentation/src/main/resources/pages/ditto/basic-connections.md [hunk 3]
```diff
@@ -116,6 +116,10 @@ only send data to the Thing `sensor:temperature1` the following enforcement conf
 }
 ```
 
+Note: This example assumes that there is a valid user named `inbound-auth-subject` in ditto.
+If you want to use a user for the basic auth (from the [HTTP API](connectivity-protocol-bindings-http.html)) use the prefic `nginxg:`, e.g. `nginx:my-basic-auth-user`.
```
Comment (thjaeckle) [inline documentation/src/main/resources/pages/ditto/basic-connections.md]:
```suggestion
If you want to use a user for the basic auth (from the [HTTP API](connectivity-protocol-bindings-http.html)) use the prefix `nginx:`, e.g. `nginx:ditto`.
```

File: documentation/src/main/resources/pages/ditto/basic-connections.md [hunk 4]
```diff
@@ -116,6 +116,10 @@ only send data to the Thing `sensor:temperature1` the following enforcement conf
 }
 ```
 
+Note: This example assumes that there is a valid user named `inbound-auth-subject` in ditto.
```
Comment (thjaeckle) [inline documentation/src/main/resources/pages/ditto/basic-connections.md]:
```suggestion
Note: This example assumes that there is a valid user named `ditto:inbound-auth-subject` in Ditto.
```

File: documentation/src/main/resources/pages/ditto/connectivity-manage-connections.md [hunk 5]
```diff
@@ -15,6 +15,12 @@ All connection related piggyback commands use the following HTTP endpoint:
 POST /devops/piggyback/connectivity
 ```
 
+## Authorization
+
+When creating new connections , an `authorizationContext` is needed which gives the _authorization subjects_ (think of authorized users) under which permissions the operations that are accepted via this connection are executed.
```
Comment (thjaeckle) [inline documentation/src/main/resources/pages/ditto/connectivity-manage-connections.md]:
```suggestion
When creating new connections, an `authorizationContext` is needed providing the _authorization subjects_ (e.g. IDs of authorized users) to use in order to determine the permissions for when the connection executes commands (e.g. modifying a Thing).
```

File: documentation/src/main/resources/pages/ditto/connectivity-manage-connections.md [hunk 6]
```diff
@@ -15,6 +15,12 @@ All connection related piggyback commands use the following HTTP endpoint:
...
+If you want to use a user for the basic auth (from the [HTTP API](connectivity-protocol-bindings-http.html)) use the prefic `nginxg:`, e.g. `nginx:my-basic-auth-user`.
```
Comment (thjaeckle) [inline documentation/src/main/resources/pages/ditto/connectivity-manage-connections.md]:
```suggestion
If you want to use a user for the basic auth (from the [HTTP API](connectivity-protocol-bindings-http.html)) use the prefix `nginx:`, e.g. `nginx:ditto`.
```

File: documentation/src/main/resources/pages/ditto/connectivity-protocol-bindings-mqtt.md [hunk 7]
```diff
@@ -57,6 +57,10 @@ For an MQTT connection:
 }
 ```
 
+Note: This example assumes that there is a valid user named `inbound-auth-subject` in ditto.
```
Comment (thjaeckle) [inline documentation/src/main/resources/pages/ditto/connectivity-protocol-bindings-mqtt.md]:
```suggestion
Note: This example assumes that there is a valid user named `ditto:inbound-auth-subject` in Ditto.
```

File: documentation/src/main/resources/pages/ditto/connectivity-protocol-bindings-mqtt.md [hunk 8]
```diff
@@ -57,6 +57,10 @@ For an MQTT connection:
...
+If you want to use a user for the basic auth (from the [HTTP API](connectivity-protocol-bindings-http.html)) use the prefic `nginxg:`, e.g. `nginx:my-basic-auth-user`.
```
Comment (thjaeckle) [inline documentation/src/main/resources/pages/ditto/connectivity-protocol-bindings-mqtt.md]:
```suggestion
If you want to use a user for the basic auth (from the [HTTP API](connectivity-protocol-bindings-http.html)) use the prefix `nginx:`, e.g. `nginx:ditto`.
```

**Comment by JulianFeinauer** [comment]
@thjaeckle thanks for the review and sorry for the wanky formulations sometimes. I updated all your suggestions.

File: documentation/src/main/resources/pages/ditto/basic-auth.md [hunk 9]
```diff
@@ -33,3 +33,13 @@ Authorization is implemented with an <a href="#" data-toggle="tooltip" data-orig
 (in API version 2).
 
 Please find details at [ACL](basic-acl.html) and [Policies](basic-policy.html).
+
+### Authorization Context in DevOps Commands
+
+An `authorizationContext` which is passed to [DevOps Commands](installation-operating.html#devops-commands) DevOps commands needs
```
Comment (ffendt) [inline documentation/src/main/resources/pages/ditto/basic-auth.md]: There's a little duplication here: 

> passed to [DevOps Commands](...) DevOps commands

Comment (JulianFeinauer) [reply documentation/src/main/resources/pages/ditto/basic-auth.md]: Thank you @ffendt , fixed it!

**Pull request merged**